docs/guide/search.md

```diff
@@ -569,70 +678,64 @@ ezpublish:
-- Running out of memory during indexing
-    - In general make sure to run indexing using prod environment to avoid debuggers and loggers from filing up memory.
-    - Stash: Disable in\_memory cache as recommended on [Persistence cache](repository.md#persistence-cache) for long running scripts.
-    - Flysystem: You can find further info in: <https://jira.ez.no/browse/EZP-25325>.
-
 The last step is to execute the initial indexation of data:
 
 ``` bash
 php app/console --env=prod --siteaccess=<name> ezplatform:solr_create_index
 ```
 
-SOLR BUNDLE &gt;= 1.2Since eZ Platform v1.7.0 the `ezplatform:solr_create_index` command is deprecated, use `php app/console ezplatform:reindex` instead:
+##### Possible exceptions
 
-``` bash
-php app/console --env=prod --siteaccess=<name> ezplatform:reindex
-```
+If you have not configured your setup correctly, some exceptions might happen on indexing.
+Here are the most common issues you may encounter:
 
-### Configuring the Solr Search engine Bundle
+- Exception if Binary files in database have an invalid path prefix
+    - Make sure `var_dir` is configured properly in `ezplatform.yml` configuration.
+    - If your database is inconsistent in regards to file paths, try to update entries to be correct *(make sure to make a backup first)*.
+- Exception on unsupported Field Types
+    - Make sure to implement all Field Types in your installation, or to configure missing ones as [NullType](field_type_reference.md#null-field-type) if implementation is not needed.
+- Content is not immediately available 
+    - Solr Bundle on purpose does not commit changes directly on Repository updates *(on indexing)*, but lets you control this using Solr configuration. Adjust Solr's `autoSoftCommit` (visibility of changes to search index) and/or `autoCommit` (hard commit, for durability and replication) to balance performance and load on your Solr instance against needs you have for "[NRT](https://cwiki.apache.org/confluence/display/solr/Near+Real+Time+Searching)".
+- Running out of memory during indexing
+    - In general make sure to run indexing using the prod environment to avoid debuggers and loggers from filling up memory.
+    - Stash: Disable inMemory cache as recommended in [Persistence cache](repository.md#persistence-cache) for long-running scripts.
+    - Flysystem: You can find further info in https://jira.ez.no/browse/EZP-25325.
 
-For configuration of Solr connection for your repository, see [How to set up Solr Search engine](#how-to-set-up-solr-search-engine) above.
+### Configuring the Solr Search Engine Bundle
```

Comment:
search engine bundle vs Search Engine Bundle - lets use one version in the doc (not only Solr case)

Reply:
I've been thinking of using "Solr search engine" (this is a description, not a proper name -> even Solr itself doesn't use this), and Solr Search Engine Bundle (becuase our bundle names are capitalized). Do you think it's too confusing?

Reply:
In 557 we have: 'The Solr search engine bundle' in other places we have e.g. 'Legacy Search Engine Bundle' with all words capitalized, I wanted consistent naming convention for bundles only. 
As all bundles are capitalized let's stick with it and I'm good with leaving 'Solr search engine' in lower case.

Reply:
You're right, way to go being consistent, me ;)
Fixed.